src/main/resources/application.properties

```diff
@@ -0,0 +1,18 @@
+logging.level.root=INFO
+
+database.name=${DB_NAME:signal-hub}
+database.host=${DB_HOST:localhost}
+database.port=${DB_PORT:5432}
+database.username=${DB_USER:postgres}
+database.password=${DB_PASSWORD:postgres}
+
+#R2DBC - Postgres
+spring.r2dbc.url=r2dbc:postgresql://${database.host}:${database.port}/${database.name}
+spring.r2dbc.username=${database.username}
+spring.r2dbc.password=${database.password}
+spring.r2dbc.pool.max-size=20
+spring.r2dbc.pool.initial-size=5
+spring.r2dbc.pool.enabled=true
+spring.data.r2dbc.repositories.enabled=true
+
+pdnd.history-cleanup.delay-days=30
```

Comment:
delay days configuration should be injected as env variable

Reply:
abbiamo provveduto ad iniettare la variabile di ambiente sulla parte infrastrutturale